content/docs/tutorials/introduction.md

```diff
@@ -157,10 +156,13 @@ Contributions are welcome! Please refer to [CONTRIBUTING.md](./CONTRIBUTING.md)
 ### Companies using Coraza
 
 * [Babiel](https://babiel.com) (supporter)
```

Comment:
Is this still true?

Reply:
Nope, let's limit the list to the formal stuff we know are using coraza. Do we need permissions from companies or we should just list them ?

Reply:
To list then we don't need permissions. We can link to documentation or other stuff we deem necessary.